releases/release-7.1.0.md

```diff
@@ -187,6 +274,25 @@ In v7.1.0, the key new features and improvements are as follows:
 
     Starting from v7.1.0, when providing the query service of [`INFORMATION_SCHEMA.TIFLASH_TABLES`](/information-schema/information-schema-tiflash-tables.md) and [`INFORMATION_SCHEMA.TIFLASH_SEGMENTS`](/information-schema/information-schema-tiflash-segments.md) system tables for TiDB, TiFlash uses the gRPC port instead of the HTTP port, which avoids the security risks of the HTTP service.
 
+* Support LDAP authentication [#43580](https://github.com/pingcap/tidb/issues/43580) @[YangKeao](https://github.com/YangKeao) **tw:ran-huang**
+
+    Starting from v7.1.0, TiDB supports LDAP authentication and provides two authentication plugins: `authentication_ldap_sasl` and `authentication_ldap_simple`.
+
+    For more information, see [documentation](/security-compatibility-with-mysql.md).
+
+* Refactor the database auditing feature (Enterprise Edition) **tw:hfxsd**
+
+    In v7.1.0, the database auditing capabilities have been refactored for TiDB Enterprise Edition, which significantly increases the scope of capabilities and improves the experience to meet the needs of enterprises for database security compliance:
+    
+    - Introduce the concepts of "Filter" and "Rule" to provide more granular audit event definitions and support more fine-grained audit settings.
+    - Support defining rules in JSON format, providing a more friendly way for configuration.
+    - Add automatic log rotation and space management functions. Support configuring log rotation in two dimensions: retention time and log size.
+    - Support outputting audit logs in both TEXT and JSON formats for easy integration with third-party tools.
+    - Support audit log redaction. You can replace all literals to enhance security.
+    
+  Database auditing is an important feature in TiDB Enterprise Edition. Database auditing provides a powerful monitoring and auditing tool for enterprises to ensure data security and compliance. It can help managers in enterprises track the source and impact of database operations to ensure that data would not be illegally stolen or tampered with. At the same time, database auditing can also help enterprises to comply with various regulatory and compliance requirements to safeguard the legal and ethical compliance of enterprises. This feature has a very important application value for enterprise information security.
+    
+   This is feature is included in TiDB Enterprise Edition. Go to [TiDB Enterprise page](https://www.pingcap.com/tidb-enterprise/) to get TiDB Enterprise Edition and its documentation.
```

Comment:
```suggestion
* Enhance the database auditing feature (Enterprise Edition) **tw:hfxsd**

    In v7.1.0, TiDB Enterprise Edition enhances the database auditing feature, which significantly expands its capacity and improves the user experience to meet the needs of enterprises for database security compliance:

    - Introduce the concepts of "Filter" and "Rule" for more granular audit event definitions and more fine-grained audit settings.
    - Support defining rules in JSON format, providing a more user-friendly configuration method.
    - Add automatic log rotation and space management functions, and support configuring log rotation in two dimensions: retention time and log size.
    - Support outputting audit logs in both TEXT and JSON formats, facilitating easier integration with third-party tools.
    - Support audit log redaction. You can replace all literals to enhance security.
    
  Database auditing is an important feature in TiDB Enterprise Edition. This feature provides a powerful monitoring and auditing tool for enterprises to ensure data security and compliance. It can help enterprises managers in tracking the source and impact of database operations to prevent illegal data theft or tampering. Furthermore, database auditing can also help enterprises meet various regulatory and compliance requirements, ensuring the legal and ethical compliance. This feature has important application value for enterprise information security.
    
   This feature is included in TiDB Enterprise Edition. To use this feature and its documentation, navigate to [TiDB Enterprise page](https://www.pingcap.com/tidb-enterprise/).

```